installer/dev/install.cpp

```diff
@@ -156,20 +194,24 @@ namespace ProjectReunionInstaller {
 
         // Add the package
         Uri packageUri{ packageFilename };
-        hresult hrAddResult = AddPackage(packageUri);
+        auto hrAddResult = AddPackage(packageUri, packageProperties);
         if (!quiet)
         {
-            std::wcout << "Package deployment result : 0x" << std::hex << hrAddResult.value << std::endl;
+            std::wcout << "Package deployment result : 0x" << std::hex << hrAddResult << std::endl;
```

Comment:
Blech. printf is so much more dense and easier to read. Alas, no [std::format](https://en.cppreference.com/w/cpp/utility/format/format) until C++20. But wprintf will be easier than cout to transition to std::format so I'd recommend it even today